governance.md

```diff
@@ -0,0 +1,162 @@
+# Django Commons Governance
+
+For Django Commons to function effectively, we need trusted volunteers to step up and take responsibility for certain aspects that require ongoing attention and a certain level of trust from our community. These volunteers are organized into teams, each with a specific focus.
+
+## Relation to the Django Software Foundation
+
+Django Commons is an independent initiative from the Django Software Foundation (DSF). We are here for the third-party packages that compliment Django. We are here as a community for and by Django users.
+
+## Teams
+
+There are four teams that work together to handle the operations of Django Commons. 
+
+They are:
+
+- Admin Team
+- CoC Team
+- PyPI Team
+- Security Team
+- Communication
```

Comment:
```suggestion
```
I believe this was a heading from the other file that was pulled in accidentally.

Reply:
Thanks for catching that! You were right -- fixed.